.github/workflows/run-k8s-tests-on-sev.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI | Run kubernetes tests on SEV
+on:
+  workflow_call:
+    inputs:
+      registry:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+
+jobs:
+  run-k8s-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu-sev
+    runs-on: sev
```

Comment:
@ryansavino do we have already the SEV machine added on github as a self-hosted runner?

Reply:
Yep, both a rome and a milan have been added for sev, sev-es and sev-snp respectively.